extensions/wikia/Recirculation/js/recirculation.js

```diff
@@ -1,12 +1,13 @@
 /*global define*/
 define('ext.wikia.recirculation.recirculation', [
 	'jquery',
+	'wikia.window',
 	'wikia.abTest',
 	'wikia.tracker',
 	'wikia.nirvana',
 	'videosmodule.controllers.rail',
 	'ext.wikia.adEngine.taboolaHelper'
-], function ($, abTest, tracker, nirvana, videosModule, taboolaHelper) {
+], function ($, w, abTest, tracker, nirvana, videosModule, taboolaHelper) {
```

Comment:
Won't adding a param in the beginning break something else elsewhere? (backwards compatibility)